src/lib/parse_org.js

```diff
@@ -634,6 +635,10 @@ export const parseOrg = fileContents => {
               default: false,
             })
           );
+        } else {
+          if (line.startsWith('#+')) {
```

Comment:
I'd combine this with the previous line into an `if else`

Reply:
Done and changed further